skimage/measure/simple_metrics.py

```diff
@@ -29,31 +24,34 @@ def compare_mse(im1, im2):
 
     Parameters
     ----------
-    im1, im2 : ndarray
-        Image.  Any dimensionality.
+    im_true : ndarray
+        Ground-truth image, same shape as im_test.
+    im_test : ndarray
+        Test image.
 
     Returns
     -------
     mse : float
         The mean-squared error (MSE) metric.
 
     """
-    _assert_compatible(im1, im2)
-    im1, im2 = _as_floats(im1, im2)
-    return np.mean(np.square(im1 - im2), dtype=np.float64)
+    warn('DEPRECATED: skimage.measure.compare_mse has been moved to '
+         'skimage.evaluation.mean_squared_error. It will be removed from '
+         'skimage.measure in version 0.18.')
+    return mean_squared_error(im1, im2)
 
 
-def compare_nrmse(im_true, im_test, norm_type='Euclidean'):
+def compare_nrmse(im_true, im_test, norm_type='euclidean'):
```

Comment:
Call `.lower()` to do not break legacy user's codes.

Reply:
right, `.lower` gets called in the corresponding function in evaluate, and `compare_nrmse` calls it

https://github.com/scikit-image/scikit-image/blob/7bf1da54cd1f91514da7ab6ac7482e9ce64d26c9/skimage/evaluate/simple_metrics.py#L80